lighthouse-extension/app/src/popup.js

```diff
@@ -53,7 +53,7 @@ document.addEventListener('DOMContentLoaded', _ => {
     qsBody += '**Stack Trace**:\n ```' + err.stack + '```';
 
     const base = 'https://github.com/googlechrome/lighthouse/issues/new?';
-    const title = encodeURI('title=Lighthouse Extension Error');
+    const title = encodeURI('title=Extension Error: ' + err.message);
     const body = '&body=' + encodeURI(qsBody);
```

Comment:
Mocking this up in Chrome, Github seems to break around 77 characters per line (60 characters of this error message). 

Are two lines okay, or is one line preferable?
<img width="1078" alt="screen shot 2016-11-21 at 11 57 00 am" src="https://cloud.githubusercontent.com/assets/1470297/20492144/a764b646-afe1-11e6-9a97-107e6b94c936.png">